src/test/java/com/puppycrawl/tools/checkstyle/checks/metrics/ClassFanOutComplexityCheckTest.java

```diff
@@ -371,4 +371,18 @@ public void testClearStatePackageName() throws Exception {
                 .isTrue();
     }
 
+    /**
+     * Test [ClassName]::new expression. Such expression will be processed as a
```

Comment:
What does this have to do with the pattern array converter? All the properties in the file are default which means it doesn't go against the converter, right?

Reply:
This has nothing to do with the converter. Please see https://github.com/checkstyle/checkstyle/pull/14740#issuecomment-2047109647.

This is a part of the workaround for `ClassFanOutCheck` until #14787

TLDR;
I have to keep the following line, otherwise the CI will fail.
https://github.com/checkstyle/checkstyle/blob/7d5d4d21e3a139ec80120f02e08def8d347f9911/src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/AbstractClassCouplingCheck.java#L121
Added this test case to kill Pitest mutation for this line. After #14787 is solved, we will not need this anymore.